app/sessionStore.js

```diff
@@ -252,7 +252,12 @@ module.exports.cleanAppData = (data, isShutdown) => {
   if (data.sites) {
     const clearHistory = isShutdown && getSetting(settings.SHUTDOWN_CLEAR_HISTORY) === true
     if (clearHistory) {
+      // TODO - this should the history methods from siteUtils
       data.sites = data.sites.filter((site) => site && site.tags && site.tags.length)
+      data.sites = data.sites.map((site) => {
+        delete site.lastAccessedTime
```

Comment:
The clear modal and context menu in about:history set the value to null, but this works too.  Verified by cherry picking into my branch and manually testing :+1: